test/Npgsql.Tests/Types/EnumTests.cs

```diff
@@ -8,11 +8,11 @@
 
 namespace Npgsql.Tests.Types
 {
-    class EnumTests : TestBase
+    public class EnumTests : TestBase
     {
         public EnumTests(string backendVersion) : base(backendVersion) {}
 
-        enum Mood { Sad, Ok, Happy };
+        public enum Mood { Sad, Ok, Happy };
```

Comment:
Same here, you can make this internal in order to reference it from FunctionTests